etna/transforms/outliers/base.py

```diff
@@ -197,7 +197,7 @@ def _inverse_transform(self, df: pd.DataFrame) -> pd.DataFrame:
         return df
 
     @abstractmethod
-    def detect_outliers(self, ts: TSDataset) -> Dict[str, List[pd.Timestamp]]:
+    def detect_outliers(self, ts: TSDataset) -> Dict[str, pd.Series]:
```

Comment:
Shouldn't we change return signature in all other places too?

Reply:
It has already changed